arm/Microsoft.ServiceFabric/clusters/.bicep/nested_applicationTypes.bicep

```diff
@@ -0,0 +1,22 @@
+param applicationTypeObj object
+param clusterName string
+param location string
+param tags object
+param properties object
+
+resource applicationType 'Microsoft.ServiceFabric/clusters/applicationTypes@2021-06-01' = {
+  name: '${clusterName}/applicationTypes-${applicationTypeObj.name}'
+  location: location
+  tags: tags
+  properties: properties
+
+  resource versions 'versions@2021-06-01' = [for applicationTypesVersion in applicationTypeObj.applicationTypesVersions: {
```

Comment:
Must be refactored to a child resource folder & template. See for example storage account from upstream main for reference

Reply:
Refactored.